app/models/reviewable_history.rb

```diff
@@ -4,16 +4,22 @@ class ReviewableHistory < ActiveRecord::Base
   belongs_to :reviewable
   belongs_to :created_by, class_name: 'User'
 
-  def self.types
```

Comment:
Here I refactored our custom enum by using the one that is shipped with `ActiveRecord`. I think we should probably migrate all our enums but I guess this should be discussed :grin:

Reply:
I think this is not a bad idea but probably should be a separate PR.

Reply:
How does this version handle addition / deletion from the list? We manually assign id/number to each values because we tend to add/remove elements later on.

Reply:
Actually it’s quite simple, by default it will just use `0` for the first element and increment from there. But if we want to change that, it’s just a matter of providing the values we want:
```ruby
enum status: {
  pending: 0,
  approved: 1,
  rejected: 25,
  ignored: 42,
  deleted: 99
}
```
We can even provide strings or symbols if we were using PG enums :slightly_smiling_face: